lib/index.js

```diff
@@ -78,45 +80,50 @@ export const decorateFooter = (Footer, { React, PropTypes }) => {
         price,
         updateFeed,
         updateFiat,
+        hideMenus = false,
         refreshPrice
       } = this.props;
 
       const Ticker = () => (
-        <div className={'col'} style={{height: '100%', lineHeight: '200%'}}>
+        <div className={'col'} style={{ height: '100%', lineHeight: '200%' }}>
           <Text>
-            {price.crypto +
-              ' ' +
+            {'1 ' + price.crypto +
+              ' = ' +
               price.fiatSymbol +
               moneyNumber(price.price) +
               ' '}
           </Text>
-          <select
-            value={price.fiat}
-            onChange={e => {
-              updateFiat(e.target.value);
-              refreshPrice();
-            }}
-          >
-            {price.availableFiats.map(val => (
-              <option value={val} key={val}>
-                {val}
-              </option>
-            ))}
-          </select>
-          &nbsp;
-          <select
-            value={price.feed}
-            onChange={e => {
-              updateFeed(e.target.value);
-              refreshPrice();
-            }}
-          >
-            {price.availableFeeds.map(val => (
-              <option value={val} key={val}>
-                {val}
-              </option>
-            ))}
-          </select>
+          {!hideMenus && (
```

Comment:
The only issue I see with this, is that not all currencies have a symbol in front of the value. So this menu is the only indicator what currency you are on. Might be fine without? Or maybe just leave the text,  but not a menu?

Reply:
I guess this is what prompted me for the update above. I could make the arrangement/styles above conditional on if there is a menu or not.